### ISSUE TITLE
Changing bifaciality factor values does not modify annual energy

### ISSUE THREAD

**Issue by mimxvi** [NREL/pysam#18]
Hello,

NREL-PySAM: 2.0.2
NREL-PySAM-stubs: 2.0.2
Installed in: 5.4.15-2-MANJARO
With: conda version : 4.7.12
With: python version : 3.7.4

Problem: When changing the bifaciality factor values the final annual energy does not change.

Detailed description: I set up a PV plant configuration with SAM. Then, I export the parameters in a json file. In a python script, I load the parameters and execute. When I change the bifaciality factor and execute again the resulting annual energy is always the same. If I modify other parameters, like the ground_clearance_height then the annual energy changes, as expected. In no case, the annual energy calculated with the python script is equal to that calculated directly with SAM (it is similar but different). It seems like some parameter is not being used in the simulation with PySAM and also that the bifacility factor is ignored.

Example:
```
import json
import PySAM.Pvsamv1 as sampv
from PySAM.PySSC import dict_to_ssc_table

# Loading predefined plant with SAM in a json
# 065 in file-name means "Bifaciality factor = 0.65"
with open('PV-plant_bifaciality-065.json') as sg_plant_params_file:
    plant_dict = json.load(sg_plant_params_file)
reference_pvplant_065 = dict_to_ssc_table(plant_dict, "Pvsamv1")
plant065 =  sampv.wrap(reference_pvplant_065)

# Print loaded attributes
print('module_model (0=SPE): ',
      plant065.Module.module_model)
print('spe_is_bifacial: ',
      plant065.SimpleEfficiencyModuleModel.spe_is_bifacial)
print('spe_bifaciality: ',
      plant065.SimpleEfficiencyModuleModel.spe_bifaciality)
print('spe_bifacial_transmission_factor: ',
      plant065.SimpleEfficiencyModuleModel.spe_bifacial_transmission_factor)
print('spe_bifacial_ground_clearance_height: ',
      plant065.SimpleEfficiencyModuleModel.spe_bifacial_ground_clearance_height)

# Execute
plant065.execute()
print('annual energy (with Bf 0.65): ', plant065.Outputs.annual_energy, '[kWh]')

# Modify Bifaciality factor and execute again
plant065.SimpleEfficiencyModuleModel.spe_bifaciality = 0.80
plant065.execute()
print('annual energy (with Bif_f 0.80): ', plant065.Outputs.annual_energy, '[kWh]')

# make sure the attribute is right
print('spe_bifaciality: ',
      plant065.SimpleEfficiencyModuleModel.spe_bifaciality)

# try modify other parameter: from ground_clearance_height 1 to 1.5
plant065.SimpleEfficiencyModuleModel.spe_bifacial_ground_clearance_height = 1.5
plant065.execute()
print('annual energy (with GCH 1.5): ', plant065.Outputs.annual_energy, '[kWh]')

# try deactivate bifacial
plant065.SimpleEfficiencyModuleModel.spe_is_bifacial = 0
plant065.execute()
print('annual energy (with Bif 0): ', plant065.Outputs.annual_energy, '[kWh]')
```
The result:
_[mimxvi scripts]$ python -u testing_sam.py 
module_model (0=SPE):  0.0
spe_is_bifacial:  1.0
spe_bifaciality:  0.6499999761581421
spe_bifacial_transmission_factor:  0.013000000268220901
spe_bifacial_ground_clearance_height:  1.0
annual energy (with Bf 0.65):  2595990.586107735 [kWh]  # This value directly in SAM is 2537343[kWh].
annual energy (with Bif_f 0.80):  2595990.586107735 [kWh]
spe_bifaciality:  0.8
annual energy (with GCH 1.5):  2622277.451320002 [kWh]
annual energy (with Bif 0):  2415502.2998500825 [kWh]_

I do not know what I am doing wrong, or what is happening. In the web, it is said that bifaciality factor is in %, but even changing one order of magnitude does not help.
"spe_bifaciality
float: Bifaciality factor [%]
Required: set to 0 if not provided."

Thank you very much in advance for any help.

**Comment by dguittet** [comment]
Thanks for reporting this bug! I have tested and can confirm that this is an issue in SAM in general. The bug is located in https://github.com/NREL/ssc in the files related to cmod_pvsamv1, so I'll be moving the discussion there, linking to this. We'll be looking at fixing this and re-releasing a new SAM and PySAM version within the month.

**Comment by cwhanse** [comment]
@dguittet when you have a change to track this bug down, it would help to know what SAM has been doing, to inform already-published comparisons among bifacial irradiance models.

**Comment by dguittet** [comment]
@cwhanse Yes, good point! Adding @janinefreeman @cpaulgilman who may have insight on bifacial model changes.

**Comment by janinefreeman** [comment]
@cwhanse- this bug hasn't been present in previous released versions of SAM that were used to perform those published comparisons, this is something that was introduced in this Beta version. Hopefully that limits its impact!

**Comment by mimxvi** [comment]
Thank you very much for your answers.
I'm going to move to a previous version for the moment.
Thanks for your extraordinary and valuable effort in the development of this amazing tool.

**Comment by janinefreeman** [comment]
Confirmed that this bug was introduced only in this Beta version, not in any previous releases, so should not be present in any published comparisons of this model with others. Fixed in NREL/ssc@ab65852b9464cd94f6c09caf67d9ec7a7717d50b
